### PR TITLE
Document macOS Monterey Support

### PR DESCRIPTION
**Pull request by fkorotkov**
And deprecate Catalina

### CODE REVIEW

File: docs/guide/macOS.md [hunk 1]
```diff
@@ -5,14 +5,26 @@ Use `macos_instance` in your `.cirrus.yml` files:
...
   script: echo "Hello World from macOS!"
 ```
 
 ### List of available images
 
+#### macOS Monterey
+
+* `monterey-base` - vanilla macOS with Brew and Command Line Tools pre-installed.
+* `monterey-xcode-NN` - based of `catalina-base` with Xcode NN and couple other packages pre-installed: 
+  `cocoapods`, `fastlane`, `rake` and `xctool`. [Flutter](https://flutter.dev/) and Android SDK/NDK are also pre-installed.**
```
Comment (edigaryev) [inline docs/guide/macOS.md:20]:
```suggestion
  `cocoapods`, `fastlane`, `rake` and `xctool`. [Flutter](https://flutter.dev/) and Android SDK/NDK are also pre-installed.
```

Comment (edigaryev) [reply docs/guide/macOS.md:20]: The same thing also exists in the "macOS Big Sur" section for some reason.

Comment (fkorotkov) [reply docs/guide/macOS.md]: That's weird. See a2edaa5

Comment (fkorotkov) [reply docs/guide/macOS.md]: Added in 14a4302

**Review by edigaryev — APPROVED** [review]

**Pull request merged**